lib/errors.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+/**
+ * @module Errors
+ */
+/**
+ * Factory functions to create throwable error objects
+ */
+
+/**
+ * @summary
```

Comment:
`@summary` is _implied_. You only really need to specify this tag if you also provide a description.